ckanext/canada/plugins.py

```diff
@@ -245,6 +247,12 @@ def before_search(self, search_params):
             search_params['extras']['imso_approval'] = u'true'
             search_params['fq'] += '+ready_to_publish:"true", +imso_approval:"true", -portal_release_date:*'
 
+        # CKAN Core search view wraps all fq values with double quotes.
+        # We need to remove double quotes from the portal_release_date queries.
```

Comment:
do they also escape values inside the quotes that we might need to worry about?

Reply:
Does not look like anything is escaped: https://github.com/ckan/ckan/blob/master/ckan/views/dataset.py#L193

Reply:
that's concerning

Reply:
Haha upstream issue??

Reply:
yes to me it says "I'm just praying there aren't any any quotes or other characters that need special handling here"